gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -443,6 +445,12 @@ protected void configureDialog(DirectoryDialog dialog) {
         requiredFieldMessage.setForeground(getDisplay().getSystemColor(SWT.COLOR_RED));
         requiredFieldMessage.setVisible(false);
 
+        gpuProfilingCapabilityWarning = withLayoutData(
+            createLabel(this, ""),
```

Comment:
`createLabel(this, NO_GPU_PROFILING_CAPABILITY)`

and remove the setText below. `setText` typically requires a relayout, you are just getting lucky that it wasn't needed.